.gitignore

```diff
@@ -0,0 +1 @@
+*.crx
```

Comment:
@ravenjohn haha, I'm demonstrating how to give feedback on Github for one of our guy. You may want to unsubscribe this project as it no longer used. I'm rewriting this widget for HB